mods/e2140/content/ed/infantry/a02/rules.yaml

```
@@ -0,0 +1,41 @@
+ed_infantry_a02:
+	Inherits@1: ^EdInfantry
+	Inherits@2: ^CoreArmedSelfInfantry
+	Tooltip:
+		Name: A02 Android
```

Use original name *Android A02*:

```suggestion
		Name: Android A02
```